admin-commands/raid-cleanup.md

```diff
@@ -20,12 +20,12 @@ This command cleans up any raid invasion by banning all users who joined after a
 
 ### Flags
 
-* `--clean-messages` - Setting this flag will delete the messages sent by the banned users in the last 7 days.
+* `clean-messages` - Setting this flag will delete the messages sent by the banned users in the last 7 days.
```

Comment:
Flags turned into arguments and should be documented as such (will write this only once)

Reply:
I think I've caught most of these. Just doing a sweep to make sure